romancal/ramp_fitting/ramp_fit_step.py

```diff
@@ -194,8 +193,14 @@ def create_image_model(input_model, image_info):
     meta = dict(wcs=None)  # default empty WCS
     meta.update(input_model.meta)
     meta["cal_step"]["ramp_fit"] = "INCOMPLETE"
-    meta["cal_logs"] = maker_utils.mk_cal_logs()
-    meta["photometry"] = maker_utils.mk_photometry()
+    meta["cal_logs"] = rds.CalLogs()
+    meta["photometry"] = rds.Photometry(
+        {
+            "conversion_megajanskys": -999999,
```

Comment:
```suggestion
            "conversion_megajanskys": -999999,
```
Eventually these could be `None`. However if we set them to `None` now files produced by this step that aren't later run through "photom" will cause resample to raise an exception due to the lack of support of blending `None` values: https://github.com/spacetelescope/roman_datamodels/issues/406

As far as I can tell this doesn't break any current regtests (although several report compare against truth differences as these values are `None` instead of -999999).

If it's preferred to change this to `None` and okify the differences let me know.

Reply:
FWIW, I'm okay with having a garbage sentinel value in here (-9999...).  None is fine too; on the one hand, maybe it's nice that it's more likely to cause crashes later on; on the other hand, maybe someone knowingly messed with the units of the file and isn't helped by also needing to mess with the units here.  So let's leave this as you have it.